tests/test_dummy.py

```diff
@@ -0,0 +1,8 @@
+# pylint: disable=C0114
```

Comment:
This is a dummy test file. This file should be removed as soon as we have real test files.

Reply:
Great! do you have a tutorial on the format of the tests?

Reply:
Not at the moment, but I will look for one during the weekend :)